eventuals/then.h

```diff
@@ -95,16 +90,14 @@ struct _Then final {
                     std::is_void_v<Arg>,
                     std::invoke_result<F_>,
                     std::invoke_result<F_, Arg>>::type,
-                void,
-                Errors>>;
-
+                void>>;
```

Comment:
```suggestion
                void,
                std::tuple<>>>;
```
And we also need to pass `std::tuple<>` in `ValueFromMaybeComposable` above too!